src/Microsoft.Framework.Logging.Testing/LogValuesAssert.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright(c) Microsoft Open Technologies, Inc.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Xunit.Sdk;
+
+namespace Microsoft.Framework.Logging.Testing
+{
+    public static class LogValuesAssert
+    {
+        /// <summary>
+        /// Asserts that all the expected values are present in the actual values by ignoring
+        /// the order of values.
+        /// </summary>
+        /// <param name="expectedValues">Expected subset of values</param>
+        /// <param name="actualValues">Actual set of values</param>
+        /// <param name="comparer">
+        /// An <see cref="IEqualityComparer{T}"/> to compare the values. 
+        /// In case no comparer is provided, a default comparer is used.
+        /// </param>
+        public static void Contains(
+            IEnumerable<KeyValuePair<string, object>> expectedValues,
```

Comment:
Yeah, this seems generic which is not exactly for log values, but I did not want to have something like `Contains(ILogValues expected, ILogValues actual)` as individual tests need to construct the ILogValues object which is more work.
OR
Should I create a custom type like the below and expect all tests to use this? we can then change this Contains method parameters to accept ILogValues instead?
```c#
public class CustomLogValues : Dictionary<string, object>, ILogValues
{
    public IEnumerable<KeyValuePair<string, object>> GetValues()
    {
        return this;
    }
}

//-------------

var expectedLogValues = new CustomLogValues();
expectedLogValues.Add("RouteKey", "id");
expectedLogValues.Add("ActionId", "Blah");

//-------------

LogValuesAssert.Contains(expectedLogValues, actualLogValues);
```